File: URI/C/TempoDeJogoComMinutos.c
#include <stdio.h>

int main() {
    int inicioH, inicioM, fimH, fimM, durH = 0, durM = 0;
    scanf("%d %d %d %d", &inicioH, &inicioM, &fimH, &fimM);
    if (inicioH == fimH && inicioM == fimM)
    {
        durH = 24;
    }
    
    while (inicioH != fimH && inicioM != fimM)
    {
        inicioH++;
        durH++;

        if (inicioH == 24)
        {
            inicioH = 0;
        }
    }
    durM = fimM - inicioM;
    if (durM < 0)
    {
        durH--;
        durM += 60;
    }
    
    printf("O JOGO DUROU %d HORA(S) E %d MINUTO(S)\n", durH, durM);

 
    return 0;
}

File: URI/Python/TempoDeJogoComMinutos.py
inf = [int(x) for x in input().split()]
inicioH = inf[0]
inicioM = inf[1]
fimH = inf[2]
fimM = inf[3]
durH = 0
durM = 0
if (inicioH == fimH and inicioM == fimM):
    durH = 24

while (inicioH != fimH and inicioM != fimM):
    inicioH+=1
    durH+=1
    if (inicioH == 24):
        inicioH = 0

durM = fimM - inicioM
if (durM < 0):
    durH-=1
    durM += 60

print("O JOGO DUROU {} HORA(S) E {} MINUTO(S)".format(durH, durM))
